digital_land/commands.py

```diff
@@ -715,9 +737,91 @@ def add_data(
 
     if user_response != "yes":
         print("Operation cancelled by user.")
-        clear_log(collection_dir, endpoint_resource_info["endpoint"])
         return
 
+    if not cache_dir:
+        cache_dir = Path("var/cache/add_data/")
+    output_path = os.path.join(
+        cache_dir, "transformed", endpoint_resource_info["resource"] + ".csv"
+    )
+    issue_dir = os.path.join(cache_dir, "issue/")
```

Comment:
could use the Path class from pathlib instead of os, it's a more OOP focussed library and can make some of this a bit easier

Reply:
I've changed these dirs to be Path objects now